content/neuvector-prime/azure/contents.lr

```diff
@@ -169,6 +169,19 @@ Once logged in, you can begin to [navigate and configure NeuVector]. Refer (http
 
    !["Keyvaluepair config"](./keyvaluepairconfig.png)
 
+
+## Install Scanner Updater
+
+The NeuVector scanner image is updated daily with a new CVE database on the NeuVector docker hub registry.It is recommended to install scanner updater AFTER successfull initial NeuVector deployment.
+
+Execute the following command in a Cloud Shell
+
+```
+helm repo add
```

Comment:
its really just `helm repo add` with no parameters or anything???

Reply:
waiting for final command.. we will finalize tomorrow.

Reply:
this is not applicable now.